pyproject.toml

```diff
@@ -0,0 +1,98 @@
+[project]
+name = "nbgrader"
+description = "A system for assigning and grading notebooks"
+license = {file = "LICENSE"}
+authors = [{name = "Jupyter Development Team", email = "jupyter@googlegroups.com"}]
+keywords = ["Jupyter", "Notebooks", "Grading", "Homework"]
+classifiers = [
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+
+    "Framework :: Jupyter",
+    "Framework :: Jupyter :: JupyterLab",
+    "Framework :: Jupyter :: JupyterLab :: 3",
+    "Framework :: Jupyter :: JupyterLab :: Extensions",
+    "Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt",
+]
+urls = {Homepage = "https://github.com/jupyter/nbgrader"}
+requires-python = ">=3.7"
+dependencies = [
```

Comment:
Do we need the duplication here is the same information is already available in `setup.cfg`?

Reply:
:+1: